### ISSUE TITLE
Request: filter out closed milestones

### ISSUE THREAD

**Issue by jpschewe** [pockethub/PocketHub#624]
It would be really nice if when editing an issue closed milestones were not visible.

**Comment by hzsweers** [comment]
Could you give some more info? Are you talking about in the "Select Milestone" dialog?

**Comment by jpschewe** [comment]
Yes, when selecting a milestone for an issue I would like to filter out closed milestones.

**Comment by hzsweers** [comment]
I see. Yes this would be useful, though maybe as a setting and not a default.

**Comment by jpschewe** [comment]
A setting would be fine too. I have a lot of closed milestones in my projects and on the phone it gets annoying to need to scroll through a page or two of closed milestones.

This could also apply to the select milestone dialog when filtering issues as well.

**Comment by hzsweers** [comment]
Yeah that's understandable. Couple routes we can take as far as applying a filter. This won't be done until after the first release though, just FYI. Focusing primarily on a solid base and migration first.
